app/controllers/warehouse_reports/health/enrollments_controller.rb

```diff
@@ -71,8 +72,36 @@ def download
       end
     end
 
+    def override
+      @file = Health::Enrollment.find(params[:id].to_i)
+      @transactions = Kaminari.paginate_array(@file.transactions).page(params[:page])
+      receiver_id = @file.receiver_id
+      @receiver = @receiver = Health::Cp.find_by(
+        pid: receiver_id[0...-1],
+        sl: receiver_id.last,
+      )
+      errors = @file.processing_errors
+
+      overrides = override_params[:overrides].map { |p| p.match(/ID (\d*)/).try(:[], 1) }.compact
```

Comment:
Let's put this into a method (it can be in the controller) `medicaid_ids_from_form_results(override_params[:overrides])`